aldryn_news/views.py

```diff
@@ -59,6 +59,7 @@ def get_context_data(self, **kwargs):
         kwargs['year'] = int(self.kwargs.get('year')) if 'year' in self.kwargs else None
         if kwargs['year']:
             kwargs['archive_date'] = datetime.date(kwargs['year'], kwargs['month'] or 1, 1)
+        kwargs['tags'] = Tag.objects.all()
```

Comment:
@fbyaguar Hi! do we need all tags? I saw the logic for showing only those tags that are used in news